src/pages/Intro.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
 
-import { Routes } from '@/constants';
 import Page from '@/layouts/Page';
+import PlaceholderImage from '@/assets/placeholder.svg';
+
+const heroStyle = {
+  backgroundImage: `url(${PlaceholderImage})`
+}
 
 const Intro = () => (
   <Page>
-    <div className="intro">
+    <Row className="intro__hero p-5 mb-5" style={heroStyle}>
```

Comment:
Row is only used for creating grids. Should be followed by Col. Suggest using regular block elements
```suggestion
    <div className="intro__hero p-5 mb-5" style={heroStyle}>
```